src/Core/Utilities/Fido2/Fido2ClientException.cs

```diff
@@ -0,0 +1,25 @@
+namespace Bit.Core.Utilities.Fido2
+{
+    public class Fido2ClientException : Exception
+    {
+        public enum ErrorCode 
+        {
+            NotAllowedError,
+            TypeError,
+            SecurityError,
+            UriBlockedError,
+            NotSupportedError,
+            InvalidStateError,
+            UnknownError
+        }
+
+        public readonly ErrorCode Code;
+        public readonly string Reason;
```

Comment:
Fixed! I never really got why everyone loves properties :)

Reply:
In this particular case just to be consistent to other places, but they have some perks like you can set a breakpoint in the property and it will stop on the getter to see the value (and the stack trace) whereas you cannot do that with the field. Also I like the reasons explained [here](https://softwareengineering.stackexchange.com/questions/161303/is-it-bad-practice-to-use-public-fields) although in this case we could have used fields given that it's very simple and they are not likely to change or used in Reflection.